packages/pluto/build.rs

```diff
@@ -0,0 +1,13 @@
+use std::path::PathBuf;
+use std::process::{exit, Command};
+
+fn main() -> Result<(), std::io::Error> {
+    let root_json_path = PathBuf::from(env!("PUBLISH_REPO_ROOT_JSON"));
+    println!("cargo:rerun-if-changed={}", root_json_path.display());
```

Comment:
This should not be a build time dependency. What is going on here? The package should use the same `build.rs` as all the other packages.

Reply:
I am not sure why i copied this out of os, it is a mistake, i shall strike it from the record haha